_config.php

```diff
@@ -24,24 +24,9 @@
         'friendly_name' => 'Default CMS',
         'priority' => '50',
         'skin' => 'silverstripe',
-        'body_class' => 'typography',
```

Comment:
Why are we removing this?

Reply:
That's in the set of default settings already. I removed everything that is in the default settings so that changing the default settings will update for this as well.